<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSpecsWindow</class>
 <widget class="QWidget" name="SystemSpecsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="SystemPara">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>581</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>System parameters</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="mag_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>System magnifications</string>
    </property>
    <property name="text">
     <string>Magnification</string>
    </property>
   </widget>
   <widget class="QLabel" name="NA_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Numerical Aperture</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
   </widget>
   <widget class="QLabel" name="pix_size_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera pixel size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="upsam_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Upsample rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="NA_list_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Illumination angle list</string>
    </property>
    <property name="text">
     <string>Angle list</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mag">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>optional</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NA">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>optional</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pix_size">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>optional</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="upsam">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NA_list">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(Read Only)</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="GPU_ID_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>-1 for no GPU, otherwise 0,1,2,3 or ...</string>
    </property>
    <property name="text">
     <string>GPU ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GPU_ID">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="pix_size_unit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(um)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Wavelength">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Illumination angle list</string>
    </property>
    <property name="text">
     <string>Wavelength</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lambda">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>optional</string>
    </property>
   </widget>
   <widget class="QLabel" name="lambda_unit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(um)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="AlgcomboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>221</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Gerchberg-Saxton</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EPRY Embedded pupil function recovery</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gauss-Newton</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KK Kramers-Kronig relation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>APIC Angular Pychographic Imaging with Closed-form method</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="SltAlgorithm_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Algorithm selection</string>
    </property>
    <property name="text">
     <string>Algorithm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cfm_sys_butt">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="ROI_slt_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Algorithm selection</string>
    </property>
    <property name="text">
     <string>ROI selection</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ROIsltbox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[1,1,256,256]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
